EventFilter/L1GlobalTriggerRawToDigi/test/L1GtPacker_cfg.py

```diff
@@ -276,7 +276,7 @@
 # load and configure modules via Global Tag
 # https://twiki.cern.ch/twiki/bin/view/CMS/SWGuideFrontierConditions
 
-process.load('Configuration.StandardSequences.Geometry_cff')
+process.load("Configuration.StandardSequences.GeometryDB_cff")
 process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
 
 process.GlobalTag.globaltag = useGlobalTag+'::All'
```

Comment:
just wanted to mention that all the `::All` here are relics from Cond DB version 1 and they certainly cannot work with the current DB infrastructure.